DOC-2017: Added TINY-9757 to the release notes. (#2711)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

{productname} 6.5 also includes the following improvements:

> === Now `icon` field for dialog footer `togglebutton`s is not mandatory.
> //#TINY-9757
> 
> In previous versions of {productname} it was mandatory for the `DialogFooterToggleButton` to have an icon prop. As a consequence the `DialogFooterToggleButton` would not function without the icon.
> 
> Improvements were made in {productname} 6.5, that now allows the icon prop to be optional.

Reviewer: ```suggestion
=== `DialogFooterToggleButton` icon prop is now optional
//#TINY-9757

Previously it was mandatory for the `DialogFooterToggleButton` to have an icon prop.

Toggle buttons in dialog footers did not function without a displayed icon.

As of {productname} 6.5 the icon prop is now optional and toggle buttons in dialog footers work with and without a displayed icon.
```
